### ISSUE TITLE
Can't recognize items 

### ISSUE THREAD

**Issue by redhell** [tommasomarchionni/homebridge-openHAB#8]
Hi,

first of all thanks for this plugin!

The plugin can't recognize items when there is only one element in a frame.
Example:
 {
            "widgetId": "rwe_0",
            "type": "Frame",
            "label": "Eingang",
            "icon": "frontdoor",
            "widget": {
                "widgetId": "rwe_0_0",
                "type": "Text",
                "label": "Eingang [zu]",
                "icon": "frontdoor-closed",
                "item": {
                    "type": "ContactItem",
                    "name": "rweContact8d7f9f",
                    "state": "CLOSED",
                    "link": "http://192.168.2.118:8080/rest/items/rweContact8d7f9f"
                }
            }
        },

but when i add an text element the sensor will be found

{
        "id": "rwe",
        "title": "Zuhause",
        "link": "http://192.168.2.118:8080/rest/sitemaps/rwe/rwe",
        "leaf": "false",
        "widget": [{
            "widgetId": "rwe_0",
            "type": "Frame",
            "label": "Eingang",
            "icon": "frontdoor",
            "widget": [{
                "widgetId": "rwe_0_0",
                "type": "Text",
                "label": "Eingang [zu]",
                "icon": "frontdoor-closed",
                "item": {
                    "type": "ContactItem",
                    "name": "rweContact8d7f9f",
                    "state": "CLOSED",
                    "link": "http://192.168.2.118:8080/rest/items/rweContact8d7f9f"
                }
            }, {
                "widgetId": "rwe_0_0_1",
                "type": "Text",
                "label": "Test",
                "icon": "info"
            }]
        },

**Comment by Oggerschummer** [comment]
Can confirm this is an issue.
I created 1 item in e.g. homebridge.items, only referencing one simple single knx-bound switch.
Added this into sitemap homebridge.sitemap, only containing this item.

REST service did show the item properly in the sitemap, so I was confident it would work.
After some hours with reinstallation down to node.js the item was still not showing up as accessory.
Simple adding a second switch in the sitemap/items definition did the trick, things are working fine now.
Maybe a simple index thing...
